src/Connector.SqlServer/Features/IBuildCreateContainerFeature.cs

```diff
@@ -11,7 +12,8 @@ public interface IBuildCreateContainerFeature
     {
         IEnumerable<SqlServerConnectorCommand> BuildCreateContainerSql(ExecutionContext executionContext,
```

Comment:
Do not change a public interface.

There are three options here:

Go with a breaking change (Not allowed).
Default implementation (Not popular as it hides in the IDE's).
Extend the interface (I would do this, however any type checks would need to be updated).

Also SanitizedSqlName should be a string with the sanitization performed elsewhere.

Reply:
With regards to all these `IBuild*Feature` and `DefaultBuild*Feature` I don't really see them being used or possibility to be used as public extension points. At least without major hazzles or changes in SqlConnector code.
Maybe you have some examples of such usings?

Because for me at first look, they could be made `internal` without any issues.